changelog/fragments/envtest-bump.yaml

```diff
@@ -0,0 +1,12 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bumped the K8s_VERSION and envtest version to 1.24 in the Makefile
+    kind: "change"
+    breaking: false
+
+    migration:
+      header: Bumped the envtest and k8s version makefile variables to 1.24
+      body: |
+        Updated Makefile variables and testdata envtest k8s version to 1.24
```

Comment:
we only add changelog when the change affects the users.
To change the projects which are generated with SDK it needs to be changed in the Kubebuilder and then we do the bump here, see: https://github.com/kubernetes-sigs/kubebuilder/pull/2713

You only change the version in the Makefile of SDK project which is used to show the info when we run operator-sdk version command

Reply:
@camilamacedo86 Thanks, wasn't sure if I should add it or not. Removed the changelog.

Reply:
We can only update this file when we bump KB using this ENVTEST version
and bump k8s 1.24 on SDK.
It should be done as a follow up of : https://github.com/operator-framework/operator-sdk/pull/5814

Reply:
We will also update the java plugin with https://github.com/operator-framework/operator-sdk/pull/5814#issuecomment-1149186886